cmd/skupper/skupper.go

```diff
@@ -209,6 +221,12 @@ installation that can then be connected to other skupper installations`,
 	cmd.Flags().StringVarP(&routerCreateOpts.Password, "console-password", "", "", "Skupper console user. Valid only when --console-auth=internal")
 	cmd.Flags().BoolVarP(&routerCreateOpts.ClusterLocal, "cluster-local", "", false, "Set up skupper to only accept connections from within the local cluster.")
 
+	cmd.Flags().BoolVarP(&routerCreateOpts.IsEdge, "edge", "", false, "Configure as an edge")
+	f := cmd.Flag("edge")
+	f.Deprecated = "This flag is deprecated, use --router-mode [interior|edge]"
+	f.Hidden = true
+	cmd.Flags().StringVarP(&routerMode, "router-mode", "", "edge", "Skupper router-mode")
```

Comment:
iirc, router-mode should default to "interior" in anticipation of edge flag going away.

Suggest the router-mode arg be verified as one of "interior" or "edge" to detect bad value

Reply:
all true!, fixed in the last commit.

Thanks!